server/code/setup/cpdb

```diff
@@ -243,9 +274,9 @@ if __name__ == "__main__":
             print("ERROR: Please provide a password for the Candlepin database user.")
             sys.exit(1)
 
-    if (not options.create and not options.update) or \
+    if (not options.create and not options.update and not options.validate) or \
         (options.create and options.update):
```

Comment:
Unless I'm misreading the logic here would allow for "--validate --create".  Which I think would be okay, but we'd need to move the `if options.validate` portion up to the top to insure the order is correct.

Reply:
Fixed as per online discussion